src/main/scala/dev/atedeg/Extensions.scala

```diff
@@ -0,0 +1,30 @@
+package dev.atedeg
+
+import scala.language.higherKinds
+import scala.util.Try
+
+import better.files.File
+import cats.Traverse
+import cats.syntax.all._
+
+object Extensions {
+
+  @SuppressWarnings(Array("org.wartremover.warts.Equals"))
+  implicit final class AnyOps[A](self: A) {
+    def ===(other: A): Boolean = self == other
+  }
```

Comment:
Since `cats` is a dependency of the project, why you don't use this `===` from [cats](https://typelevel.org/cats/typeclasses/eq.html)? Using the cats' `===` you could remove the suppress warning.